src/Umbraco.Web/Models/MediaCropDataSet.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Dynamic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Web;
+using Newtonsoft.Json;
+using Umbraco.Core;
+using Umbraco.Core.Dynamics;
+using Umbraco.Core.Models;
+using Umbraco.Core.Serialization;
+using Umbraco.Web.PropertyEditors.ValueConverters;
+
+namespace Umbraco.Web.Models
+{
+    [JsonConverter(typeof(NoTypeConverterJsonConverter<MediaCropDataSet>))]
+    [TypeConverter(typeof(MediaCropDataSetConverter))]
+    [DataContract(Name="mediaCropDataSet")]
+    public class MediaCropDataSet : ImageCropDataSet, IHtmlString, IEquatable<MediaCropDataSet>
+    {
+        [DataMember(Name = "udi")]
+        public Udi Udi { get; set; }
```

Comment:
Not sure about naming this `Udi`, maybe just `Id` would be better?

Reply:
But it's a `Udi` :)

I don't feel right picking between the two, if HQ wants it to be `Id`, then it's fine with me.